thavalon-webapp/src/components/Account.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import React, { ChangeEvent } from 'react';
 
 export function Account() {
+    const profileImage = React.useRef<HTMLImageElement>(null);
+    function handleImageUpload(event: ChangeEvent<HTMLInputElement>): void {
+        // get the chosen file. There should be only 1 because multiple choices disabled.
+        const file = event.target.files;
+        if (file === null || file.length !== 1) {
+            return;
+        }
+        const fileItem = file.item(0);
+        if (fileItem === null) {
+            return;
+        }
+
+        let imageElement: HTMLImageElement | null = profileImage.current;
+        const reader = new FileReader();
+        reader.onload = (event: ProgressEvent<FileReader>) => {
+            // verify not accessing null elements on anything
+            if (imageElement === null || event.target === null || event.target.result === null) {
```

Comment:
nit: can we condense this as `event.target?.result`?

Reply:
Account page was a stub/test, I'm just going to replace it with just an h1 for now